pharmd-app/src/components/Fields/ScoredListField/ScoredListField.js

```diff
@@ -1,12 +1,10 @@
 import React from "react";
 import { useGetOne } from "react-admin";
-
-import ScoredListItem from "./ScoredListItem";
 import List from "@material-ui/core/List";
+import ScoredListItem from "./ScoredListItem";
 
 const ScoredListField = ({ record = {}, source }) => {
-  let courses = record[source];
-  console.log("COURSES", courses);
+  const courses = record[source];
   return (
     <List>
       {courses.map((course, index) => {
```

Comment:
Is this already an issue that exists or is fixed?

Reply:
` // TODO: Need to fix render issue here `
`const { data, loading, error } = useGetOne("courses", course.course_id);`